.storybook/theme/StoryWrapper.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
I read somewhere this yields different results behind the curtain but I haven't learned about it, do you know what the difference is? 🤔

Reply:
IIRC this is the recommended way of importing React

From https://github.com/facebook/react/pull/18102
```
This is the correct way to import React from an ES module since the ES module will not have a default export. Only named exports.
```